bokehjs/src/coffee/models/ranges/range1d.coffee

```diff
@@ -18,14 +18,6 @@ export class Range1d extends Range
       max = Math.max(@_initial_start, @_initial_end)
       @setv({bounds: [min, max]}, {silent: true})
 
-  constructor: () ->
-    # new Range1d({start: start, end: end}) or Range1d(start, end)
-    if this instanceof Range1d
-      return super(arguments...)
-    else
-      [start, end] = arguments
-      return new Range1d({start: start, end: end})
-
```

Comment:
I am not in favor of removing this, but if you insist, since this will break people's code a migration note is in order

Reply:
This shorthand syntax was possible in ES5, but it's not possible in ES6. ES6 classes can't be initialized without `new` and this trick doesn't work anymore. TypeScript will reject this as well. This can be restored in future by implementing an API layer on top of models, but that will require much more work than could spend in this PR.

Reply:
no problem, as I said it just needs a migration note

Reply:
I added a migration note.